ABloom/ABloom/Presentation/SignUp/SignUpViewModel.swift

```diff
@@ -17,6 +17,8 @@ final class SignUpViewModel: ObservableObject {
   @Published var isCheckedPrivacyPolicy = false
   @Published var isCheckedTermsOfuse = false
   @Published var isCheckedOverFourteen = false
+  @Published var isCheckedSensitiveInfo = false
+  @Published var isAllChecked = false
```

Comment:
```suggestion
   var isAllChecked: Bool {
      isCheckedTermsOfuse && isCheckedOverFourteen && 
      isCheckedPrivacyPolicy && isCheckedSensitiveInfo
    }
```

이렇게 계산(연산)프로퍼티를 사용하면 checkAllAgreementAreChecked 메서드 없이 변수를 사용해 볼 수 있을 것 같아요-!!

Reply:
오! 저도 배워갑니다!